apps/valkyrie/mix.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Valkyrie.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :valkyrie,
+      version: "0.2.3",
+      elixir: "~> 1.8",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      aliases: aliases(),
+      test_paths: test_paths(Mix.env()),
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        coveralls: :test,
+        "coveralls.detail": :test,
+        "coveralls.post": :test,
+        "coveralls.html": :test
+      ]
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Valkyrie.Application, []}
+    ]
+  end
+
+  defp deps do
+    [
+      {:benchee, "~> 1.0", only: [:integration]},
+      {:brook, "~> 0.4"},
+      {:cachex, "~> 3.1"},
+      {:checkov, "~> 0.4.0", only: [:test]},
+      {:credo, "~> 1.0", only: [:dev, :test], runtime: false},
+      {:distillery, "~> 2.1"},
+      {:divo, "~> 1.1", only: [:dev, :test, :integration]},
+      {:divo_kafka, "~> 0.1", only: [:integration]},
+      {:divo_redis, "~> 0.1", only: [:integration]},
+      {:excoveralls, "~> 0.11.1", only: :test},
+      {:husky, "~> 1.0", only: :dev, runtime: false},
```

Comment:
can you remove husky as part of this migration?

Reply:
done